metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -68,6 +72,60 @@
 # patch the implementation.
 
 
+def bigquery_audit_metadata_query_template(
+    dataset: str, use_date_sharded_tables: bool
```

Comment:
@varunbharill the newly updated method should support both date sharded and partitioned tables